docs/blog/newsletter-april-2025.rst

```diff
@@ -0,0 +1,125 @@
+
+See more Write the Docs resources about `docs-as-code </topics/#docs-as-code>`__.
+
+-----------------------------------
+Leveling up as a solo documentarian
+-----------------------------------
+ 
+Working alone as a documentarian and feeling stuck in your current role? You’re not alone. Many solo documentarians find themselves in workplaces that don’t prioritize documentation, leaving them with limited opportunities to expand their skills. If you’re in this position, don’t wait for growth opportunities – create them! Here’s some ideas on how from `#career-advice <https://writethedocs.slack.com/archives/C6ADX1YVA>`__:  
+
+- Seek out problems and solve them  
+  - Identify gaps in existing documentation or areas that need more clarity.  
+  - Talk to customer support or engineers to uncover frequent pain points.  
+  - Turn these insights into FAQs, internal knowledge bases, or process docs.  
+- Build your own portfolio  
+  - Even if your leadership doesn’t prioritize documentation, write it anyway.  
+  - Create deep-dive guides, whitepapers, or best practices—whether or not someone requested them.  
+  - Treat it as a way to showcase your skills for future roles.  
+- Go beyond writing  
+  - Develop standards and style guides to improve documentation quality.  
+  - Mentor non-writers on best practices to create a stronger writing culture.  
+  - Consider branching into related roles like product management or UX writing.  
+- Expand your learning outside work  
+  - Take online courses on API documentation, structured authoring, or AI in tech communications.  
+  - Contribute to open-source projects to gain real-world experience.  
+  - Get involved in professional communities such as Write the Docs.  
+
+If you love your workplace but crave more growth, lean into the skills you can develop. But if stagnation is holding you back, don’t be afraid to explore new opportunities. Your career is yours to shape!
+
+See more Write the Docs resources about `career growth </topics/#career-growth>`__.
+
+-------------------------------
+SEO for docs: How much and how?
+-------------------------------
+
+A question was recently raised about whether technical documentation could benefit from search-engine optimization (SEO) or whether to push back against people requesting it.
+
+In general, respondents agreed that it was worth doing SEO work within reason. In fact, one person noted that keeping docs within general standards of clarity, structure, and accessibility is already a good start for many SEO practices. Another noted this may be enough for small collections of docs, but larger ones have more of a need for SEO. It's important to make sure your docs are findable and useful to your customers.
```

Comment:
```suggestion
In general, respondents agreed that it was worth doing SEO work within reason. In fact, one person noted that keeping docs within general standards of clarity, structure, and accessibility is already a good start for many SEO practices. Another noted this may be enough for small collections of docs, but larger ones have more of a need for SEO. It's important to make sure your content is findable and useful to your customers.
```